heat-stack/app/utils/hooks/use-rules-engine.ts

```diff
@@ -63,131 +59,6 @@ export const useRulesEngine = (
 			})
 	}
 
-	/** RECALCULATE WHEN BILLING RECORDS UPDATE -- maybe this can be more generic in the future */
-	const recalculateFromBillingRecordsChange = (
```

Comment:
I just wanted to take this moment to thank `recalculateFromBillingRecordsChange()` for everything it has given us 🥲